Product/Production/Services/CORE_X12DocumentSubmissionCore/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/genericbatch/request/inbound/AbstractInboundCORE_X12DSGenericBatchRequest.java

```diff
@@ -1,36 +1,38 @@
 /*
- * Copyright (c) 2014, United States Government, as represented by the Secretary of Health and Human Services.
- * All rights reserved.
+ * Copyright (c) 2009-2015, United States Government, as represented by the Secretary of Health and Human Services. 
+ * All rights reserved. 
```

Comment:
Is your IDE set to trim trailing spaces?  These were added into the license across several files.